File: PatientPortal/src/app/_models/template.ts
export class Template {
    id: string;
    companyId: string;
    templateOrder: number;
    name: string;
    title: string;
    reportTitle: string;
    detailedTemplateHtml: string;
    initialDetailedTemplateHtml: string;
    defaultTemplateHtml: string;
    isRequired: boolean;
    isActive: boolean;
    isHistorical: boolean;
    templateTypeId: string;

    constructor() {
        this.isActive = true;
        this.isRequired = false;
        this.isHistorical = false;
    }
}

File: api/Medico.Api/DB/Models/Template.cs
using System;
using System.Collections.Generic;

namespace Medico.Api.DB.Models
{
    public class Template : BaseActiveEntity
    {
        public Guid? CompanyId { get; set; }

        public Company Company { get; set; }

        public int? TemplateOrder { get; set; }

        public string Name { get; set; }

        public string Title { get; set; }

        public string ReportTitle { get; set; }

        public string Value { get; set; }

        public string DetailedTemplateHtml { get; set; }

        public string DefaultTemplateHtml { get; set; }

        public bool IsRequired { get; set; }

        public bool? IsHistorical { get; set; }

        public Guid TemplateTypeId { get; set; }

        public TemplateType TemplateType { get; set; }

        public List<ChiefComplaintTemplate> ChiefComplaints { get; set; }
    }
}
